aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/endpoints/impl/TestEndpoints.java

```diff
@@ -0,0 +1,33 @@
+package aquality.tracking.integrations.core.endpoints.impl;
+
+import aquality.tracking.integrations.core.Configuration;
+import aquality.tracking.integrations.core.IHttpClient;
+import aquality.tracking.integrations.core.endpoints.ITestEndpoints;
+import aquality.tracking.integrations.core.models.Suite;
+import aquality.tracking.integrations.core.models.Test;
+import aquality.tracking.integrations.core.utilities.JsonMapper;
+
+import javax.inject.Inject;
+import java.net.URI;
+import java.util.List;
+
+public class TestEndpoints extends AqualityTrackingEndpoints implements ITestEndpoints {
+
+    private static final String CREATE_OR_UPDATE_TEST_ENDPOINT = "/api/public/test/create-or-update";
```

Comment:
just a thought: do not we want to move such endpoints into json config?

Reply:
Why? Of course not. Json config is for end user. End user won't change endpoints ever.